packages/test/test-end-to-end-tests/src/testPersistedCache.ts

```diff
@@ -0,0 +1,66 @@
+/*!
```

Comment:
@tyler-cai-microsoft Did you intend this to be a rename? I don't see a deleted file.

Reply:
That was weird, I think when I switched branches the change deleting the file was lost